prow_config.yaml

```diff
@@ -22,61 +22,16 @@ workflows:
       - kustomize-fns/*
       - testing/*
       - py/*
-  #****************************************************************************************************
-  # Per https://github.com/kubeflow/kfctl/issues/57 the following tests should not be blocking
-  # presubmits.      
+
   - py_func: kubeflow.kfctl.testing.ci.kfctl_e2e_workflow.create_workflow
-    name: kfctl-go-iap
+    name: kfctl-e2e
     job_types:
-      - postsubmit
-      - periodic
+     - presubmit
```

Comment:
Do you just need presubmit here? What about periodic?

Reply:
Right, added it